src/site/shared/utils/input/InputManager.ts

```diff
@@ -49,28 +74,7 @@ export class InputManager extends Observable<InputManagerEvent> {
     /** The canvas the user is performing inputs on. */
```

Comment:
Maybe rename `InputManager` to `CanvasEventAdapter`/`InputEventAdapter`, or something.